<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>introwindow</class>
 <widget class="QWidget" name="introwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titreDeLaSRieLabel">
       <property name="text">
        <string>Titre de la leçon :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titreDeLaSRieLineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="motFranAisLabel">
       <property name="text">
        <string>Mot français :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="motFranAisLineEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="motNErlandaisLabel">
       <property name="text">
        <string>Mot néerlandais :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="motNErlandaisLineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="numRoDeLaSRieSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="numRoDeLaSRieLabel">
       <property name="text">
        <string>Numéro de la leçon :</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Valider le mot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Valider la leçon</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
